scripts/tests/ota_test.sh

```diff
@@ -15,30 +18,28 @@ OTA_REQUESTOR_FOLDER="out/ota_requestor_debug"
 CHIP_TOOL_APP="chip-tool"
 CHIP_TOOL_FOLDER="out"
 
```

Comment:
Can we also add a "mkdir /tmp/ota/" here? If the folder does not exist the script fails in a confusing fashion.

Reply:
I added mkdir -p /tmp/ota to the .yaml file. Do you think it should be here as well?

Reply:
If the expectation is for the script to be run, independent of the git workflow, then yes... I would ideally expect the github workflow to simply call a test script which does the setup, runs the tests, and tears things down.